tests/opensearch/basic.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+
+load "$HELPERS_ROOT/test-helpers.bash"
+
+ensure_variables_set BATS_SUPPORT_ROOT BATS_ASSERT_ROOT BATS_DETIK_ROOT BATS_FILE_ROOT TEST_NAMESPACE FLUENTBIT_IMAGE_TAG OPENSEARCH_IMAGE_TAG
+
+load "$BATS_DETIK_ROOT/utils.bash"
+load "$BATS_DETIK_ROOT/linter.bash"
+load "$BATS_DETIK_ROOT/detik.bash"
+load "$BATS_SUPPORT_ROOT/load.bash"
+load "$BATS_ASSERT_ROOT/load.bash"
+load "$BATS_FILE_ROOT/load.bash"
+
+setup() {
+    echo "recreating namespace $TEST_NAMESPACE"
+    run kubectl delete namespace "$TEST_NAMESPACE"
```

Comment:
For me I think we should also have a `teardown` method which allows us to skip as well: then you can skip it when you run locally or want to debug things.

We don't want to leave all this running, particularly on a local cluster when we start running other things where it will be quickly resource limited. Also on GKE, etc. we want to kill it at the end of test.

Reply:
good it, makes sense. will fix